index.html

```diff
@@ -72,6 +76,191 @@ <h1>
+            </p>
+            <ul class="frameworks-list work-child">
+              <li class="framework">html</li>
+              <li class="framework">Ruby on rails</li>
+              <li class="framework">css</li>
+              <li class="framework">javaScript</li>
+            </ul>
+            <button type="button" class="project-btn work-child">See Project</button>
+          </div>
+        </div>
+      </section>
+      <div id="container2">
+        <section id="about-me">
+          <div id="aboutme-desc">
+            <h2 class="work-child" id="ab-me">About Myself</h2>
+            <p class="work-child" id="ab-me-text">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <h3 class="contact-p work-child">LET’S CONNECT</h3>
+            <ul class="work-child icons2 icons">
+              <li>
+                <a target="blank" href="https://twitter.com/LisandroSeia"
+                  ><img alt="twitter" src="./assets/imgs/twitter.png"
+                /></a>
+              </li>
+
+              <li>
+                <a
+                  target="blank"
+                  href="https://www.linkedin.com/in/lisandro-seia-295120225/"
+                  ><img alt="linkedin" src="./assets/imgs/linkedin.png"
+                /></a>
+              </li>
+
+              <li>
+                <a href="#"
+                  ><img alt="medium" src="./assets/imgs/medium.png"
+                /></a>
+              </li>
+
+              <li>
+                <a target="blank" href="https://github.com/lisandroseia"
+                  ><img alt="github" src="./assets/imgs/github.png"
+                /></a>
+              </li>
+
+              <li>
+                <a href="#"
+                  ><img alt="angellist" src="./assets/imgs/angellist.png"
+                /></a>
+              </li>
+            </ul>
+            <button type="button" class="project-btn work-child">Get my resume</button>
+          </div>
+          <div id="ab-me-technical">
+            <div class="technical" id="languajes">
+              <div id="lang-title">
+                <h3 class="ab-me-title">languajes</h3>
+                <i class="fas fa-chevron-down"></i>
+              </div>
+              <ul id="lang-list">
+                <li>
+                  <img alt="js" src="./assets/imgs/JS.svg" />
+                  <p>JavaScript</p>
+                </li>
+                <li>
+                  <img alt="html" src="./assets/imgs/HTML.svg" />
+                  <p>HTML</p>
+                </li>
+                <li>
+                  <img alt="css" src="./assets/imgs/CSS.svg" />
+                  <p>CSS</p>
+                </li>
+              </ul>
```

Comment:
- [ ] Looking at the figma design and yours, this section is supposed to be below the about myself section and on the same row. I think you probably have to apply  a flex column direction to this section. Check it out and fix it please.
Your design 
![image](https://user-images.githubusercontent.com/62849872/143662027-a92c09eb-fb55-4095-b9ea-c5487b7a351f.png)

Figma design 
![image](https://user-images.githubusercontent.com/62849872/143662060-9b6945df-64c4-4cec-b98f-10fb04df4638.png)

Reply:
hello, thanks for the comment, this feature is working on the project when the sizes are mobile sizes, my previous reviewers accepted that even on the previous project, what do you think I should do?

Reply:
Hi there @lisandroseia ,  your project displays well on mobile devices.  I probably didn't get the right resizing when I reviewed your project.  Sorry about that.  Please kindly submit for re-review and it will be approved.